tests/watcher/watcher.go

```diff
@@ -184,9 +184,7 @@ func (w *ObjectEventWatcher) Watch(ctx context.Context, processFunc ProcessFunc,
 				switch watchEvent.Object.(type) {
 				case *metav1.Status:
 					status := watchEvent.Object.(*metav1.Status)
-					//api server sometimes closes connections to Watch() client command
-					//ignore this error, because it will reconnect automatically
-					if status.Message != "an error on the server (\"unable to decode an event from the watch stream: http2: response body closed\") has prevented the request from succeeding" {
+					if !shouldIgnoreWatchErr(status) {
```

Comment:
Do we handle reconnects?

Reply:
yeah we had this prior to this PR, ofc I kept it in `shouldIgnoreWatchErr`